API-Curseforge/src/main/kotlin/com.github.p03w.modifold.curseforge_api/CurseforgeAPI.kt

```diff
@@ -37,6 +34,15 @@ object CurseforgeAPI : APIInterface() {
         }
     }
 
+    fun getProjectDescription(id: Int): String? {
+        return try {
+            getWithoutAuth<CurseforgeDescription>("$root/mods/$id/description").data
+        } catch (ignored: Exception) {
+            ignored.printStackTrace()
+            null
+        }
```

Comment:
When does this fail? If it's a legitimate issue the `warn` log method should probably be used as well

Reply:
It fails when the Curseforge API is down, or when the mod isn't found, which in both cases will have been logged by the curse API calls earlier I would assume

Reply:
Yeah in both cases you'd get an error back at curseforge project collection

Reply:
Well technically not found is a warning but yeah